adapters/pangle/pangle.go

```diff
@@ -16,9 +16,16 @@ type adapter struct {
 	Endpoint string
 }
 
+type NetworkIDs struct {
+	AppID       string `json:"appid,omitempty"`
+	PlacementID string `json:"placementid,omitempty"`
+}
+
 type wrappedExtImpBidder struct {
 	*adapters.ExtImpBidder
-	AdType int `json:"adtype,omitempty"`
+	AdType     int         `json:"adtype,omitempty"`
+	IsPrebid   bool        `json:"is_prebid,omitempty"`
```

Comment:
I'm curious, why does Prebid need to be flagged specially?

Reply:
Our service handles Prebid requests and other ordinary OpenRTB requests differently. I'm hoping that we could automatically distinguish that by the request itself, not by some other manual configurations.